src/js/apps/patients/patient/patient_app.js

```diff
@@ -29,6 +29,15 @@ export default SubRouterApp.extend({
     patient: PatientSidebarApp,
   },
 
+  startRoute(currentRoute) {
+    if (this.action) {
+      this.action.trigger('editing', false);
+      delete this.action;
+    }
+
+    SubRouterApp.prototype.startRoute.call(this, currentRoute);
+  },
```

Comment:
This fixes the issue without breaking anything else, but not sure if it's the correct way to do it yet. Seems like the simplest solution to me though. I think updating `startList()` and `startPatientAction()` directly will get complicated.

Reply:
Problem is `this.action` is being cached between route changes between patient dashboard and archive. Which means that the action gets an `isEditing = true` set on it regardless of the sidebar being open or not when the route is loaded.

There's an `onStop()` that does `delete this.action`. But this app isn't being stopped when navigating between patient dashboard and archive routes.

Reply:
Gotcha, thanks 👍 . Update is made.